src/main/java/tracko/model/TrackO.java

```diff
@@ -98,20 +118,22 @@ public boolean hasItem(Item item) {
      */
     public void setItem(Item target, Item editedItem) {
         requireNonNull(editedItem);
-
         items.setItem(target, editedItem);
     }
 
-    @Override
-    public ObservableList<Order> getOrderList() {
-        return orders.asUnmodifiableObservableList();
-    }
-
     @Override
     public ObservableList<Item> getInventoryList() {
         return items.asUnmodifiableObservableList();
     }
 
+    /**
+     * Returns the inventory list object of the model.
+     * @return The inventory list object of the model.
+     */
+    public InventoryList getInventoryReference() {
+        return items;
+    }
```

Comment:
Yes. This method returns an InventoryList, so that the storage toModelType() methods can take advantage of the implemented getItem() method in InventoryList. This reduces the amount of code in the storage reading methods, because the toModelType() method in JsonAdapatedItemQuantityPair will have to iterate through an ObservableList otherwise.

Reply:
I see, thanks!